lang/cem/libcc.ansi/headers/stdint.h

```diff
@@ -33,16 +33,16 @@ typedef unsigned long   uint32_t;
 #endif
 #define INT32_MAX       2147483647
 #define INT32_MIN       (-2147483648)
-#define UINT32_MAX      4294967295
+#define UINT32_MAX      4294967295U
 
-/* We only get int64_t if longs are 8 bytes. */
+/* We only get int64_t if long longs are 8 bytes. */
```

Comment:
We should probably keep the _EM_LSIZE == 8 case here for LP64 architectures which use 64 bit longs and long longs.